.github/workflows/josh-json.yml

```
@@ -0,0 +1,35 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Josh - JSON - Node.js CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Cache
+        uses: actions/cache@v2.1.3
+        with:
+          # A list of files, directories, and wildcard patterns to cache and restore
+          path: node_modules
+          # An explicit key for restoring and saving the cache
+          key: ${{ matrix.node-version }}
+      - run: npm i -g yarn
+      - run: cd josh-tests; yarn;
+      - run: cd josh-json; yarn; yarn test; yarn lint;
```

```suggestion
      - run: cd josh-tests && yarn --ignore-scripts --frozen-lockfile
      - run: cd josh-json && yarn --ignore-scripts --frozen-lockfile && yarn test && yarn lint
```
- Ignore automated scripts with `--ignore-scripts` flag.
- Use `&&` instead of `;` so it only moves to next prompt/command if successful.
- Use the `--frozen-lockfile` flag to ensure `package.json` and `yarn.lock` are in sync.